.github/workflows/build.yml

```diff
@@ -94,19 +112,95 @@ jobs:
         ./run_example.sh --dry-run simple_app_under_sev run_test
         ./run_example.sh --dry-run simple_app_under_sev setup_with_auto_policy_generation_for_SEV
 
+        ./run_example.sh --dry-run application_service
+        ./run_example.sh --dry-run application_service setup
+
+        #! Should do nothing but just emit usage messages
+        ./run_example.sh --dry-run application_service run_test
+
+        popd
+
+    - name: test-run_example-simple_app
+      run: |
         #! ---------------------------------------------------------------------
         #! This will also check that utilities programs still compile
         echo " "
         echo "******************************************************************"
         echo "* Test: Execute script to compile, build and run simple_app."
         echo "******************************************************************"
         echo " "
+        pushd ./sample_apps
+
         ./cleanup.sh
+
+        set -x
+        ps -ef | grep -E 'simpleserver|example_app.exe|run_example.sh|app_service.exe'
+        set +x
+
         ./run_example.sh simple_app
         ./cleanup.sh
 
         popd
 
+    - name: test-build-and-setup-App-Service-and-Simple-App
+      run: |
+        echo " "
+        echo "***************************************"
+        echo "* Build-and-setup Application Service "
+        echo "***************************************"
+        echo " "
+        pushd ./sample_apps
+
+        ./run_example.sh application_service setup
+
+        echo " "
+        echo "*************************************************************"
+        echo "* Build-and-setup simple_app_under_app_service/ and run_test"
+        echo "*************************************************************"
+        echo " "
+
+        echo " "
+        echo "---- simple_app_under_app_service/ setup ----"
+        echo " "
+        ./run_example.sh simple_app_under_app_service setup
+
+        #! Start Certifier Service & Application Service together first.
+        ./run_example.sh --no-cleanup application_service start
+
+        #! Allow some time for App Service to get Certified ...
+        sleep 10
+
+        #! Once Application Service has been certified, we no longer need
+        #! this Certifier Service. Kill it, so app itself can start its
+        #! own Certifier Service.
+        ps -ef | grep simpleserver
+
+        set -x
+        kill -9 $(pgrep simpleserver)
```

Comment:
FYI: @jlmucb : I tried the alternative of using `--port` options ... but somehow the combinations of arguments needed to get Certifier-specific port numbers did not, quite, work out right.

For now, I'd like to stay with this `kill` approach. And will rework once this change stabilizes in `/main`.